desc/objectives/_geometry.py

```diff
@@ -415,6 +424,9 @@ class PlasmaVesselDistance(_Objective):
         Collocation grid containing the nodes to evaluate surface geometry at.
     plasma_grid : Grid, ndarray, optional
         Collocation grid containing the nodes to evaluate plasma geometry at.
+    use_softmin: Bool, use softmin or hard min.
+    alpha: float, parameter used for softmin. The larger alpha, the closer the softmin
```

Comment:
is this needed if we're manually ensuring that alpha*arr > 1? Should we include a note about that here?

Reply:
ensureing alpha*arr>1 isn't sufficient to ensure that the softmin is accurate, I just have it there as a check bc we know for sure if alpha*arr<1, the softmin is inaccurate. Allowing alpha to be larger than that minimum threshold value lets the user ensure that the softmin is accurate/approaching the hardmin. I will include a note about that